.editorconfig

```diff
@@ -1,13 +1,17 @@
 # Stop .editorconfig files search on current file
 root = true
 
-# Unix-style newlines with a newline ending every file
 [*]
+indent_style = space
+indent_size = 2
 end_of_line = lf
-insert_final_newline = true
+charset = utf-8
 trim_trailing_whitespace = true
+insert_final_newline = true
+max_line_length = 2000
+block_comment_start = /*
+block_comment = *
+block_comment_end = */
```

Comment:
Hm, these keys aren't present in the spec as well. Any reason to add them?

Reply:
These are for `eclint` to understand block comments.
What should I do?